var/spack/repos/builtin/packages/octopus/package.py

```diff
@@ -58,17 +83,37 @@ def install(self, spec, prefix):
             'CC=%s' % spec['mpi'].mpicc,
             'FC=%s' % spec['mpi'].mpifc,
             '--enable-mpi',
-            '--with-fft-lib=-L%s -lfftw3' % spec['fftw'].prefix.lib
-            # --with-blacs=${prefix}/lib/libscalapack.dylib
-            # --with-netcdf-prefix=netcdf-fortran
+            '--with-fft-lib=-L%s -lfftw3' % spec['fftw'].prefix.lib,
+        ])
+        if '+metis' in spec:
+            args.extend([
+                '--with-metis-prefix=%s' % spec['metis'].prefix,
+            ])
+        if '+parmetis' in spec:
+            args.extend([
+                '--with-parmetis-prefix=%s' % spec['parmetis'].prefix,
+            ])
+        if '+netcdf' in spec:
+            args.extend([
+                '--with-netcdf-prefix=%s' % spec['netcdf-fortran'].prefix,
+                '--with-netcdf-include=%s' % 
+                spec['netcdf-fortran'].prefix.include,
+            ])
+        if '+arpack-ng' in spec:
+            args.extend([
+                '--with-arpack={}'.format(arpack.joined()), 
```

Comment:
The `{}` syntax is only available in Python 2.7 and above. Since we need to support Python 2.6, you'll have to use `{0}`, which works in Python 2.6 and above.